pkg/querier/http.go

```diff
@@ -103,6 +104,19 @@ func (q *Querier) LogQueryHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	userID, err := user.ExtractOrgID(ctx)
+	if err != nil {
+		writeError(err, w)
+		return
+	}
+	if _, ok := expr.(logql.LogSelectorExpr); ok {
```

Comment:
This works, but in this handle its always a lot one, see the test below :)

It's more the Range and Instant query handle that needs the check. Although I live to have the limit check here.